docs/virtual_machines/tekton_tasks.md

```diff
@@ -44,6 +44,14 @@ spec:
   featureGates:
     deployTektonTaskResources: true
 ```
+User can use this command to enable `deployTektonTaskResources` feature gate in HCO CR
+```console
+oc patch hco kubevirt-hyperconverged  --type=merge -p '{"spec":{"featureGates": {"deployTektonTaskResources": true}}}'
+```
+or in TTO CR
```

Comment:
Does this apply only to stand-alone without HCO?

Reply:
yes

Reply:
```suggestion
or in TTO CR if TTO is deployed as a stand-alone without HCO
```